content/en/hosting/kubernetes-vs-docker.md

```diff
@@ -2,48 +2,64 @@
+Medic recommends [Docker](/hosting/4.x/docker/) for most deployments. Only use [Kubernetes](/hosting/4.x/kubernetes/) for large, multi-tenant or specific hardware constrained deployments.
 
-Given all this, we recommend:
-* New, smaller production 4.x CHT deployments can use either [Docker]({{< relref "hosting/4.x/production/docker/" >}}) or [Kubernetes]({{< relref "hosting/4.x/production/kubernetes/" >}}) depending on their expected use and growth.
-* Application development for both [CHT 3.x]({{< relref "hosting/3.x/app-developer" >}}) and [CHT 4.x]({{< relref "hosting/4.x/app-developer" >}}) should use Docker Compose.
-* Production 3.x CHT deployments should use [Docker Compose]({{< relref "hosting/3.x" >}})  - Note that 3.x is [end of life]({{< relref "releases/#supported-versions" >}})  and should only be used to support existing 3.x deployments.
+Application development for both [CHT 3.x]({{< relref "hosting/3.x/app-developer" >}}) and [CHT 4.x]({{< relref "hosting/4.x/app-developer" >}}) should use Docker as well.
 
-## Kubernetes
+## Docker 
 
-Kubernetes provides advantages in managing CHT Deployments:
+Docker's has stood the test of time and has many benefits:
 
-* Resilient network across either physical or VM nodes in the cluster. This allows strong distribution of the heavy CPU and RAM loads that CouchDB can incur under heave use.
+* Easy to deploy by just downloading a compose file.
+* Trivially scales down to a 4 core laptop and [way up to a 48 core server](https://forum.communityhealthtoolkit.org/t/investigate-adding-more-shards-as-a-potential-avenue-for-improved-performance/4831). 
+* Easily integrates with existing Docker based solutions like [Watchdog](/hosting/monitoring/) or other services running in Docker.
+* Backup and restore can be achieved via simple file management via [bind mounted volumes](https://docs.docker.com/engine/storage/bind-mounts/).
+* Just enough complexity to securely and soundly run the CHT with out extra complexities of Kubernetes 
+* Natively supports one click CHT upgrades in the admin GUI
+
+Docker Compose deployment has the following services when running the CHT:
+
+* A service for each CouchDB instance.
+* A CHT API service.
+* HAProxy Healthcheck service.
+* HAProxy service.
+* Upgrade Service service.
+* A CHT Sentinel service.
+* An nginx service to act as a reverse proxy and terminate TLS connections
+
+
+## Kubernetes (k8s)
+
+{{< callout type="warning" >}}
+Only use Kubernetes on bare-metal machines with 8 cores or fewer or when dozens of CHT instances are needed in a multi-tenant deployment.
```

Comment:
> bare-metal machines with 8 cores or fewer

I'm not sure we should stipulate the number of cores here. Maybe say low-end instead?

Reply:
+1 on Diana's comment.

Additionally, I am reluctant of the use of the word "only". We can do recommendations according to deployment needs, but we shouldn't impose a tool or another.

Reply:
gotcha.  Lemme know if e7ad45ba doesn't address your feedback!